cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMaker2Cluster.java

```diff
@@ -163,23 +162,8 @@ public String getServiceAccountName() {
         List<Volume> volumeList = super.getVolumes(isOpenShift);
 
         for (KafkaMirrorMaker2ClusterSpec mirrorMaker2Cluster: clusters) {
-            KafkaMirrorMaker2Tls tls = mirrorMaker2Cluster.getTls();
-
-            if (tls != null) {
-                List<CertSecretSource> trustedCertificates = tls.getTrustedCertificates();
-    
-                if (trustedCertificates != null && trustedCertificates.size() > 0) {
-                    for (CertSecretSource certSecretSource : trustedCertificates) {
-                        String volumeName = mirrorMaker2Cluster.getAlias() + '-' + certSecretSource.getSecretName();
-                        // skipping if a volume with same Secret name was already added
-                        if (volumeList.stream().noneMatch(v -> v.getName().equals(volumeName))) {
```

Comment:
@scholzj Hello, Is it supposed to noneMatch() here or anyMatch(). Probably all the other Volume mounts are using anyMatch() so is there any reason we are using noneMatch() here?

Reply:
You need to understand that from the code. But it should skip volumes already added, so I guess noneMatch is correct?

Reply:
Well I researched a bit about it and looks both are pretty similarin logic  just one return true and other returns false